x-pack/test/saved_object_tagging/functional/tests/bulk_actions.ts

```diff
@@ -38,7 +38,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await tagManagementPage.waitUntilTableIsLoaded();
 
         const displayedTags = await tagManagementPage.getDisplayedTagNames();
-        expect(displayedTags.length).to.be(3);
+        expect(displayedTags.length).to.be.greaterThan(2);
```

Comment:
same reason, there are now 5 tags instead of 3
```
Error: expected 5 to equal 3
```

Reply:
I thought we fixed a while ago the problem of fleet assets 'polluting' the test data for all the suites by **not** installing any assets during FTR tests. Why is this still necessary then?

Reply:
I'll dig deeper, I'm not familiar that part of the code that installs `synthetics` package on startup, it would be great to exclude that from SOT tests.

Reply:
Yea, especially given we have other tests (SO management mostly) that take as granted that the data injected via the test datasets are the only existing objects during the tests. We already encountered failures / flakiness because of these auto-installed packages in the past, and I would really love to avoid encountering that again. cc @joshdover given he was part of the discussions at that time (IIRC)

Reply:
This is the only place I know of where this package gets installed automatically: https://github.com/elastic/kibana/blob/57f7a48b8dd3047461fee36061701564a4030585/x-pack/plugins/synthetics/server/routes/synthetics_service/install_index_templates.ts#L26-L28

I doubt that is getting called in this test though? As an aside, that endpoint should really be a POST and not a GET...

Reply:
The installation of `synthetics` package comes from fleet setup here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/setup.ts#L111
I found a way to fix that by loading an empty kibana es archive like we do in fleet api tests.
```
await esArchiver.load('test/functional/fixtures/es_archiver/empty_kibana');
```

Reply:
This is more a workaround that a solution (I'd like to just have fleet **NOT** install anything by default, or at least to have the option to do so for FTR tests), but I'm in PTO starting today so I won't be able to follow this PR, so it will have to do for now. I'd love you to open an issue to discuss about this problem though, if possible.

Reply:
Sure, I can open an issue. I am not sure if there is a way to disable fleet plugin in SOT tests, I'll check.

Reply:
@juliaElastic Your link didn't work for me. We shouldn't be installing any packages by default I think.

Reply:
I found a [workaround](https://github.com/elastic/kibana/pull/137184/commits/0c9f93e0a2611a99d3ff52c02300a1e751354fd7) by setting an invalid registryUrl, this prevents installing the default packages. I didn't find a better way to disable fleet plugin from SOT tests so far.